autofill/autofill-api/src/main/java/com/duckduckgo/autofill/api/urlmatcher/AutofillUrlMatcher.kt

```diff
@@ -14,11 +14,14 @@
  * limitations under the License.
  */
 
-package com.duckduckgo.autofill.api.ui.urlmatcher
+package com.duckduckgo.autofill.api.urlmatcher
 
 interface AutofillUrlMatcher {
-    fun extractUrlPartsForAutofill(originalUrl: String): ExtractedUrlParts
-    fun matchingForAutofill(visitedSite: ExtractedUrlParts, savedSite: ExtractedUrlParts): Boolean
+    fun extractUrlPartsForAutofill(originalUrl: String?): ExtractedUrlParts
```

Comment:
nit: appreciate these methods were not documented before but would be nice to document them :)

Reply:
Good suggestion. Going to pull this class apart soon as it's doing too much, so will be sure to document the output of that.